### DISCUSSION TITLE
shelly 1pm

### DISCUSSION THREAD

**Discussion by EDSTOBI** [arendst/Tasmota#16283] [Q&A] [answered]
Hello

I have times:
{"NAME":"Shelly 1PM","GPIO":[320,0,0,0,192,2720,0,0,0,0,0,224,0,4736],"FLAG":0,"BASE":18}

odet 

{"NAME":"Shelly 1PM","GPIO":[56,0,0,0,82,134,0,0,0,0,0,21,0],"FLAG":2,"BASE":18}

for the Shelly 1PM. Both seem to work although they are very different.

Can anyone explain this to me?
Has one of the codes superseded the other?
Thank you

**Comment by barbudor** [comment, marked as answer]
Yes à new function numbering scheme has been introduce sometime back
When you enter a template with the old style, tasmota automatically converts to the new one
